conda/models/environment.py

```diff
@@ -0,0 +1,142 @@
+# Copyright (C) 2012 Anaconda, Inc
+# SPDX-License-Identifier: BSD-3-Clause
+"""Conda environment data model"""
+
+from __future__ import annotations
+
+from dataclasses import dataclass, field
+from logging import getLogger
+from typing import TYPE_CHECKING
+
+from ..exceptions import CondaValueError
+
+if TYPE_CHECKING:
+    from typing import Any
+
+    from .match_spec import MatchSpec
+    from .records import PackageRecord
+
+
+log = getLogger(__name__)
+
+
+@dataclass
+class Environment:
+    # Prefix the environment is installed into (required).
```

Comment:
Small thing but I think this comment syntax is parsed magically by Sphinx in some way:

```suggestion
    #: Prefix the environment is installed into (required).
```